README.md

```diff
@@ -57,6 +57,11 @@ Then, simply run the following command:
 
 You should now be able to use riscv64-unknown-elf-gcc and its cousins.
 
+NB: Pay attention to the core model of your embedded RISC-V target. Some targets (such as Rocket Chip) 
+use the `medany` core model instead of the default `medlow`, meaning the compiler output produced by 
```

Comment:
Sorry to be pedantic but is it true to say that Rocket Chip "uses the `medany` code model`"? 
I would have thought that Rocket Chip based RISC-V implementations can be configured with different memory maps and thus may or may not require a particular code model at software compile time?
Perhaps this is referring some some specific (default?) Rocket Chip configuration or something?